app/forms/idv/finance_form.rb

```diff
@@ -10,25 +11,14 @@ class FinanceForm
 
```

Comment:
^^ can we update the `FINANCE_HTML_OPTIONS` above as well too? that way newer browsers can help enforce some of these validations, too